react-state-3/react-state-3_journal-app-form/src/App.jsx

```diff
@@ -3,14 +3,58 @@ import EntryForm from "./components/EntryForm";
 import Footer from "./components/Footer";
 import Header from "./components/Header";
 import "./App.css";
+import { useState } from "react"
 
 function App() {
+
+  const initialEntries = [
+    {
+      id: 1000,
+      date: "Feb 5, 2025",
+      motto: "We are in a state of chaos",
+      notes:
+        "Today I learned about React State. It was fun! I can't wait to learn more.",
+    },
+    {
+      id: 999,
+      date: "Feb 4, 2025",
+      motto: "Props, Props, Props",
+      notes:
+        "Today I learned about React Props. Mad props to everyone who understands this!",
+    },
+    {
+      id: 998,
+      date: "Feb 3, 2025",
+      motto: "How to nest components online fast",
+      notes:
+        "Today I learned about React Components and how to nest them like a pro. Application design is so much fun!",
+    },
+    {
+      id: 997,
+      date: "Feb 2, 2025",
+      motto: "I'm a React Developer",
+      notes: "My React-ion when I learned about React: Yay!",
+    },
+  ];
+
+  const [entries, setEntries] = useState(initialEntries)
+
+  function handleAddEntry(newEntry){
+    const date = new Date().toLocaleDateString("en-us", { dateStyle: "medium" });
+    newEntry.date = date;
+    setEntries([...entries, newEntry]);
```

Comment:
You can put the key date (and the key id: uid()) together with ...newEntry as an object at the beginning inside the array